operations/_objects.md

```diff
@@ -1,6 +1,6 @@
 # Common objects
 
-These are JSON object definitions shared by operations across the API.
+These are JSON and other object definitions shared by operations across the API.
```

Comment:
I think it should be "These are JSON object definitions and string enumerations shared by operations across the API." - what do you think?

Reply:
I think we should go with something more universal, so we don't have to update the description each time something new that do not fit the description. I've changed it to...

`These are JSON and other common object definitions shared by operations across the API.`

How you like that?

Reply:
In that case, I think it should be "These are JSON object definitions and other JSON entities shared by operations across the API."